src/Pie.js

```diff
@@ -42,15 +42,18 @@ export default class PieChart extends Component {
         bold: true,
         color: '#ECF0F1'
       }
-    },
+    }
   }
 
   color(i) {
-    let color = this.props.color || (this.props.options && this.props.options.color)
-    if (color && !_.isString(color)) color = color.color
-    let pallete = this.props.pallete || (this.props.options && this.props.options.pallete) || Colors.mix(color || '#9ac7f7')
-    return Colors.string(cyclic(pallete, i))
-  }
+    let color = this.props.options.color
+    if (Array.isArray(color)) {
+        return color[i];
```

Comment:
This has no guard against the case of more data points than color array points.
i.e. If someone has 2 colors in the array but has 3 or more data points, this will cause an error.

My suggestion would be to have the colors cycle through the array (would need to know data count to do it)
i.e. color[i%dataCount]